web/controllers/crisis_controller.ex

```diff
@@ -0,0 +1,21 @@
+defmodule App.CrisisController do
+  use App.Web, :controller
+
+  import Ecto.Query, only: [from: 2]
```

Comment:
We can remove this line as it is already imported by `use App.Web, :controller` [here](https://github.com/LDMW/app/blob/master/web/web.ex#L35)